### PR TITLE
Restore call convention compatibility in get_model

### PR DESCRIPTION
**Pull request by tcare**
A bug surfaced where first time evaluation of a model fails due to the Model constructor throwing if the model does not exist.

Looking deeper, we see that most calls to get_model expect a possible None response and check at the call site. Unfortunately we get the same WebserviceException class for a model not being found as we do a REST error or similar.

This change is a stopgap mitigation to restore compatibility with the existing callers, and compromises by allowing the model version dependent behavior to continue passing on exceptions.

In a future follow up we should settle on a convention and allow version checks to propagate failure while still giving the possibility for handling a service exception in the caller.

### CODE REVIEW

File: diabetes_regression/util/model_helper.py [hunk 1]
```diff
@@ -40,25 +40,40 @@ def get_model(
...
+                + "or excluded"  # NOQA: E501
+            )
+        tags = [[tag_name, tag_value]]
+
+    model = None
+    if model_version is not None:
+        # TODO(tcare): Finding a specific version currently expects exceptions
+        # to propagate in the case we can't find the model. This call may
+        # result in a WebserviceException that may or may not be due to the
+        # model not existing.
```
Comment (j-so) [inline diabetes_regression/util/model_helper.py:65]: Deployment isn't using this method - why WebserviceException? It should be just the training and batch scoring pipeline that uses this.

**Comment by j-so** [comment]
LGTM - can you run the batch scoring pipeline to make sure this doesn't affect that (it shouldn't, but just to be sure)?

File: diabetes_regression/util/model_helper.py (hunk 1 above)
Comment (j-so) [reply diabetes_regression/util/model_helper.py:65]: Nevermind - I see what you're saying. Can you make a new issue for this?

Comment (tcare) [reply diabetes_regression/util/model_helper.py:65]: Will do :)

**Pull request merged**